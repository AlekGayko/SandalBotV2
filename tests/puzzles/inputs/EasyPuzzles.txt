5000 r2qb1rk/ppb2p1p/2n1pPp1/B3N3/2B1P2Q/2P2R2/1P4PP/7K w - - 0 1 
5000 8/8/6p1/7k/3r2NP/B5PK/2br1R2/8 w - - 0 1 
5000 3q1rk1/ppp1nb1p/2n1B1p1/3P1p2/2P5/1PB2N1P/PQ3PP1/5RK1 w - - 0 1 
5000 4rk2/3q1p1p/1pn3p1/p1pNp3/4P3/1PP2Q2/P4PPP/2B2RK1 w - - 0 1 
5000 6qk/8/5P1p/8/8/6QP/5PP1/4R1K1 w - - 0 1 
5000 5n1r/5N2/ppp3r1/7k/3P1R1p/5P1N/PP3K2/6R1 w - - 0 1 
5000 8/R7/1p1k2bN/pP1P2p1/3K3p/7P/8/4B3 w - - 0 1 
5000 2B5/8/3K4/1p6/2k5/P4P2/1B6/N4N2 w - - 0 1 
5000 q1nrrk2/6pp/5pbb/8/8/1B6/3B1Q2/4RK2 w - - 0 1 
5000 8/3K4/2R1P3/1P1kr3/3Npb2/4P3/8/5N2 w - - 0 1 