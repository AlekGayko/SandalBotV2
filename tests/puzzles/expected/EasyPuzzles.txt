h4h7 M2
g4f6 M2
c3h8 M3
c1h6 M3
e1e8 M2
f4f5 M2
e1d2 M2
c8a6 M2
f2f6 M2
d4b3 M2